.github/new-issues-labeler.yml

```diff
@@ -27,3 +27,6 @@
 
 'bolt':
   - '/\bbolt(?!\-)\b/i'
+
+'infrastructure:commit-access-request':
```

Comment:
Can we use a shorter label? Maybe `infra:commit-access`.

Reply:
We already have the `infrastructure` label and I was trying to follow our convention of sub-labels.  I think if we do s/infrastructure/infra/ here, then we should do it for all labels.

Reply:
I'm not insisting on `infra` in particular (even though I prefer it, and renaming labels is easy), but I do have general concerns over the length. Almost all the labels we have at the moment are at least as half as short of what is proposed here, and `infrastructure` eats a lot of space as a prefix.

I can rename existing `infrastructure` labels to `infra` if that's the only blocker.

Reply:
I'll update the label to use the short name.  Note that if we change the name of existing lables without changing the names of the pr-subscribers or issues-subscriber teams.